install.sh

```diff
@@ -3,73 +3,116 @@
 # RITA is brought to you by Offensive CounterMeasures.
 # offensivecountermeasures.com
 
+_NAME=$(basename "${0}")
+_FAILED="\e[91mFAILED\e[0m"
+_SUCCESS="\e[92mSUCCESS\e[0m"
+
+#Error handling
+#Kill 0 to kill subshells as well
+trap "printf '\n[!] Installation $_FAILED!\n'; kill 0" ERR INT
 set -o errexit
+set -o errtrace
 set -o pipefail
 
-_NAME=$(basename "${0}")
-_INSDIR="/usr/local"
+# Make sure to source the latest .bashrc
+# Hack the PS1 variable to get around ubuntu .bashrc
+OLD_PS1=$PS1
+PS1=" "
+# Hack the interactive flag to get around other .bashrc's
+set -i
```

Comment:
Commenting this out lets script execution continue.  And the install seemed to work. @Zalgo2462 do you remember what this hack was for?

Reply:
Not sure if this is it or not but running the install with `sudo` but without this `set -i` line leaves `~/.rita` and `~/go` owned by root still.  Even though there are `chown` commands later on that should switch the ownership.